projects/angular/package.json

```diff
@@ -20,6 +20,6 @@
   "peerDependencies": {
     "@angular/common": "^12.0.0",
     "@angular/core": "^12.0.0",
-    "@cds/core": "^5.0.0"
+    "@cds/core": "5 - 6"
```

Comment:
no, this is equivalent to `>=5.0.0 <6.0.0` (https://github.com/npm/node-semver#advanced-range-syntax)

Reply:
I don't think that's what we want. Shouldn't it be `>=5.0.0 <7.0.0` to allow any 5.x or 6.x version? This may be a moot point if we end up copying the functions.

Reply:
Sorry, I wrote it wrong. `5 - 6` is equivalent to `>=5.0.0 <7.0.0`